### ISSUE TITLE
After selecting emails, top bar buttons not reachable by swiping

### ISSUE THREAD

**Issue by dankeck** [thunderbird/thunderbird-android#8696]
### Checklist

- [x] I have used the search function to see if someone else has already submitted the same bug report.
- [x] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.2

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 5

### Steps to reproduce

1. Start the Android TalkBack screen reader.
2. Open Thunderbird.
3. Go to Inbox.
4. Select multiple emails, either manually or using the "Select all" command. Visually, a new top bar appears with a close button, the number of emails selected, a delete button, and a button to mark as read.
5. Try to access the new top bar by swiping to it. 

### Expected behavior

- TalkBack will read the components of the top bar as each is selected.
- TalkBack will allow the user to activate the buttons in the new top bar: close, delete, and mark as read.

### Actual behavior

- TalkBack will read the components of the original top bar that is not visible: the hamburger menu, the name of the folder, and the search button, and the sort button.
- The new top bar can only be accessed using [Explore by touch](https://support.google.com/accessibility/android/answer/6006598?hl=en), if the user is able to do so.
 
![The top bar of Thunderbird showing a close button, the text "83 selected", a delete button, a button to mark mail as read, and a three-dot menu button that is highlighted](https://github.com/user-attachments/assets/033edd17-777f-4446-8e8f-3adb9af22615)
The image above shows what is visible.

![The top bar of Thunderbird showing a hamburger menu button, the word "Inbox", a search button, a sort button, and a three-dot menu button](https://github.com/user-attachments/assets/b3c912cf-9a8b-4cb3-8bc7-d52a483e8637)
The image above shows what the screen reader reads.
